Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -180,11 +182,12 @@ public class Snapshotter {
                         context.restoreGState()
                     }
 
-                    if true { // options.showsLogo
+                    if self.options.showsLogo {
```

Comment:
Capturing self here will mean the snapshotter doesn't get deallocated. Try adding a `let options = self.options` above, like we've done for style.